gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageNewIntegrationTest.java

```diff
@@ -692,7 +692,9 @@ public void listObjectInfo_includePrefix_objectInBucket() throws Exception {
   public void listObjectInfo_includePrefix_implicitDirInBucket() throws Exception {
     GoogleCloudStorage gcs = createGoogleCloudStorage(gcsOptions);
 
-    String testBucket = gcsfsIHelper.createUniqueBucket("lst-objs_incl-pfx_impl-dir-in-bckt");
+    String testBucket =
+        gcsfsIHelper.createUniqueBucket(
+            UUID.randomUUID().toString().replaceAll("-", "").substring(0, 10));
```

Comment:
Out of scope of PR that reverts hack after JUnit fix.

Reply:
yeah. Will do the cleanup.